lms/static/scripts/frontend_apps/components/test/AppRoot-test.js

```diff
@@ -50,14 +76,38 @@ describe('AppRoot', () => {
   ].forEach(({ config, appComponent }) => {
     it('launches correct app for "mode" config', () => {
       navigateTo(`/app/${config.mode}`);
-      const services = new Map();
-      const wrapper = mount(
-        <AppRoot initialConfig={config} services={services} />
-      );
+      const wrapper = renderAppRoot({ config, services: new Map() });
       assert.isTrue(wrapper.exists(appComponent));
     });
   });
 
+  it('loads config for file picker after route change', async () => {
```

Comment:
In broad strokes, I understand what's going on in this test, which is, as I understand it "the basic-lti-launch mode will load a particular configuration, and then if you navigate from there to file-picker mode, a configuration with a particular shape will then be loaded." Adding a couple more comments might not hurt, to guide the next guy along...

Reply:
Done